src/GuardClauses/GuardAgainstNullExtensions.cs

```diff
@@ -175,6 +198,30 @@ public static partial class GuardClauseExtensions
             return input;
         }
 
+#if NET5_0_OR_GREATER
+        /// <summary>
+        /// Throws an <see cref="ArgumentException" /> if <paramref name="input" /> is an empty or white space string.
+        /// </summary>
+        /// <param name="guardClause"></param>
+        /// <param name="input"></param>
+        /// <param name="parameterName"></param>
+        /// <param name="message">Optional. Custom error message</param>
+        /// <returns><paramref name="input" /> if the value is not an empty or whitespace string.</returns>
+        /// <exception cref="ArgumentException"></exception>
+        public static ReadOnlySpan<char> NullOrWhiteSpace(this IGuardClause guardClause,
+            ReadOnlySpan<char> input,
```

Comment:
I'd say make this nullable and have an overload for just `WhiteSpace`

Reply:
Something like:

```csharp
public static ReadOnlySpan<char>? NullOrWhiteSpace(this IGuardClause guardClause,
            ReadOnlySpan<char>? input,
            string parameterName,
            string? message = null)
      // ...
```
```csharp
public static ReadOnlySpan<char> WhiteSpace(this IGuardClause guardClause,
            ReadOnlySpan<char> input,
            string parameterName,
            string? message = null)
      // ...
```

Reply:
Naming was changed